examples/drupal_loadtest.rs

```diff
@@ -157,17 +152,12 @@ async fn drupal_loadtest_login(user: &GooseUser) {
                     let form_build_id = match re.captures(&html) {
                         Some(f) => f,
                         None => {
-                            user.set_failure(&mut goose.request);
-                            let error = "login: no form_build_id on page: /user page";
-                            // We choose to both log and display errors to stdout.
```

Comment:
It might be a good idea to say that user.set_failure will do that automatically instead of removing?

Reply:
yes, good idea -- fixed